assignments/arrays.js

```diff
@@ -63,32 +63,33 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}\n`);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+const lastCar = inventory[inventory.length - 1];
+console.log(`The last car is a ${lastCar.car_make} ${lastCar.car_model}\n`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for (let i of inventory) {
+  carModels.push(i.car_model);
+}
+// console.log(`The car models listed in alphabetical order are:\n${carModels.sort().join()}`);
+console.log(carModels.sort())
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
-let carYears = [];
-console.log();
+let carYears = inventory.map((car) => car.car_year);
```

Comment:
This is really elegant, well done! 
It could be even more skipping the parenthesis in your params declaration (I believe you already know this, but in functions with just 1 param you could skip the parenthesis for added cleanness).